README.md

```diff
@@ -1,3 +1,329 @@
+   This step utilizes the Maven project management tool, specifically a
+   Maven plugin called cppwrap.
+
+2. Compile this generated C++ project. This step utilizes the cross-platform
+   CMake build system.
+
+For details on executing these build steps, refer to the section for your
+specific platform below.
+
+###Build results
+
+If all goes well, the build system will:
+
+1. Generate the Bio-Formats C++ proxy classes
+2. Build the Jace C++ library
+3. Build the Java Tools C++ library
+4. Build the Bio-Formats C++ shared library
+5. Build the showinf and minimum_writer command line tools for testing the
+   functionality.
+
+Please be patient, as the build may require several minutes to complete.
+
+Afterwards, the dist/formats-bsd subdirectory will contain the following
+files:
+
+1. libjace.so / libjace.jnilib / jace.dll
+     
+     Jace shared library
+
+2. libformats-bsd.so / libformats-bsd.dylib / formats-bsd.dll
+     
+     C++ shared library for BSD-licensed readers and writers
+
+3. jace-runtime.jar
+     
+     Jace Java classes needed at runtime
+
+4. bioformats_package.jar
+     
+     Bio-Formats Java library needed at runtime
+
+5. libjtools.so / libjtools.jnilib / jtools.dll
+     
+     Java Tools shared library
+
+6. showinf / showinf.exe
+     
+     Example command line application
+
+7. minimum_writer / minimum_writer.exe
+     
+     Example command line application
+
+Items 1-4 are necessary and required to deploy Bio-Formats with your C++
+application. Item 5 (jtools) is a useful helper library for managing the Java
+virtual machine from C++, but is not strictly necessary to use Bio-Formats.
+All other files, including the example programs and various build files
+generated by CMake, are not needed.
+
+If you prefer, instead of using the bioformats_package.jar bundle, you can
+provide individual JAR files as appropriate for your application. For details,
+see [using Bio-Formats as a Java library](https://www.openmicroscopy.org/site/support/bio-formats/developers/java-library.html).
+
+##Building in Linux
+
+###Compile-time dependencies (Linux)
+
+The following directions are specific to Ubuntu Linux. Other Linux
+distributions may have similar packages available; check your package manager.
+
+To install dependencies on Ubuntu Linux, execute:
+
+
+    # install code generation prerequisites
+    sudo aptitude install maven2
+    
+    # install build prerequisites
+    sudo aptitude install build-essential cmake libboost-thread-dev
+    
+    # install Java Development Kit
+    sudo aptitude install sun-java6-jdk
+    sudo update-alternatives --config java
+
+Then select Sun's Java implementation as the system default.
+
+It may be possible to use a different Java compiler (i.e. omit the
+sun-java6-jdk package and update-alternatives step), but we have only
+tested the compilation process with Sun's Java compiler.
+
+###How to build (Linux)
+
+The following commands will generate and build the Bio-Formats C++ bindings:
```

Comment:
We have a `build` script for Unix (tested via Travis). Should we link directly to that?